ez-launch/move/Move.toml

```diff
@@ -5,6 +5,7 @@ authors = []
 
 [addresses]
 ez_launch = "_"
+minter="0x186be62b3c3e54fa01fcc4d1e147372ab24cf6dd4ec09bbf6d570f094f7f94fe"
```

Comment:
Which network is this? We would need to change this on every network deployment right? (i.e. devnet, testnet, mainnet)

Reply:
oops deleting it now, we can provide named addresses when deploying